lfc-api/LookingForConcerts/lfc_backend/views.py

```diff
@@ -734,20 +738,22 @@ def get_tags(request, search_str):
         'format' : 'json',
         'language' : ['en', 'tr'],
         'search' : query,
-        'type':'item'
+        'type'   :'item'

```

Comment:
Why is there an extra space here?

Reply:
Thought alignment would be good for readability.

Reply:
It is working like this by the way.